### DISCUSSION TITLE
Can I use CodeEditor with Vite?

### DISCUSSION THREAD

**Discussion by Chip-L** [patternfly/patternfly#5682] [Q&A]
We are doing a new project and we would like to use `CodeEditor` in the project. The project is using Vite. From looking at the documentation, there is a peer dependency for `monaco-editor-webpack-plugin`. Since Vite doesn't use webpack, can we still use this component?

**Comment by nicolethoen** [comment]
This is a great question. As of right now, the codeeditor does require webpack for implementation. There is an issue open to update the implementation to use some newer dependencies and drop the webpack requirement: https://github.com/patternfly/patternfly-react/issues/8578

definitely feel free to note your agreement in the open issue. As of right now, there is not a road map commitment to revisit the component.
